api-report/runtime-definitions.api.md

```diff
@@ -126,7 +126,7 @@ export interface IEnvelope {
 }
 
 // @public
-export interface IFluidDataStoreChannel extends IFluidRouter, IDisposable {
+export interface IFluidDataStoreChannel extends IFluidDataStoreRuntimeEntrypoint, IDisposable {
```

Comment:
could also just use `Partial<IProvideFluidHandle>` here

Reply:
actually, we can just add to `Partial<IProvideFluidHandle>` to the implementations for now, and then we don't need these changes